opensrp-anc/src/test/java/org/smartregister/anc/library/fragment/ProfileTasksFragmentTest.java

```diff
@@ -0,0 +1,75 @@
+package org.smartregister.anc.library.fragment;
+
+
+import android.app.Activity;
+import android.content.Intent;
+import android.support.v4.app.Fragment;
+import android.support.v4.view.ViewPager;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.powermock.reflect.Whitebox;
+import org.robolectric.Robolectric;
+import org.robolectric.android.controller.ActivityController;
+import org.smartregister.CoreLibrary;
+import org.smartregister.anc.library.R;
+import org.smartregister.anc.library.activity.BaseActivityUnitTest;
+import org.smartregister.anc.library.activity.ProfileActivity;
+import org.smartregister.anc.library.presenter.ProfilePresenter;
+import org.smartregister.anc.library.util.ConstantsUtils;
+import org.smartregister.anc.library.util.DBConstantsUtils;
+
+import java.util.HashMap;
+import java.util.List;
+
+public class ProfileTasksFragmentTest extends BaseActivityUnitTest {
+    private ProfileActivity profileActivity;
+    private ActivityController<ProfileActivity> controller;
+    private ProfileTasksFragment profileTasksFragment;
+    private ProfileActivity spyActivity;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'spyActivity' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=4809639)